spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { described_class.new }
+  
```

Comment:
As per comment on `menu.rb` - there is no test for `dishes` so just delete it from code.